packages/synapse-react-client/src/components/RecentPublicationsGrid/RecentPublicationsGrid.tsx

```diff
@@ -64,104 +73,122 @@ function RecentPublicationsGrid(props: RecentPublicationsGridProps) {
-                )}
-            </Box>
+          <Box>
+            {pub.values[categoryColIndex] && (
+              <Typography
+                variant="overline"
+                fontSize={'14px'}
+                sx={{
+                  backgroundColor: 'grey.300',
+                  borderRadius: '3px',
+                  padding: '4px 8px',
+                  border: 'none',
+                  lineHeight: 'initial',
+                }}
+              >
+                {pub.values[categoryColIndex]}
+              </Typography>
+            )}
             <Typography
               variant="headline2"
               color="grey.1000"
+              fontSize={'21px'}
               sx={{
                 padding: '20px 0px',
               }}
             >
               {pub.values[titleColIndex]}
             </Typography>
-            <Typography
-              variant="body1"
-              color="grey.700"
-              sx={{
-                fontStyle: 'italic',
-                fontSize: '14px',
-              }}
-            >
-              {pub.values[journalColIndex]}
-            </Typography>
-            <Typography
-              variant="body1"
-              color="grey.700"
-              sx={{ fontSize: '14px' }}
-            >
-              {pub.values[publicationDateColIndex] &&
-                formatDate(
-                  dayjs(Number(pub.values[publicationDateColIndex])),
-                  'MMMM, YYYY',
-                )}
-            </Typography>
-          </CardContent>
+            <Box display={'flex'} gap={'8px'} flexDirection={'column'}>
+              <Typography
+                variant="body1"
+                color="grey.700"
+                fontSize={'14px'}
+                fontStyle={'italic'}
+                lineHeight={1.35}
+              >
+                {pub.values[journalColIndex]}
+              </Typography>
+              <Typography
+                variant="body1"
+                color="grey.700"
+                fontSize={'14px'}
+                lineHeight={1.35}
+                paddingBottom={{ xs: 0, md: '35px' }}
+              >
+                {pub.values[publicationDateColIndex] &&
+                  formatDate(
+                    dayjs(Number(pub.values[publicationDateColIndex])),
+                    'MMMM, YYYY',
+                  )}
+              </Typography>
+            </Box>
+          </Box>
         )}
       </Box>
     </Grid>
   )
 
   return (
-    <Box
+    <Grid
       sx={{
         display: 'grid',
         gridTemplateColumns: { xs: '1fr', md: '3fr 1fr' },
-        gap: { xs: '38px', md: '80px' },
+        height: { xs: 'auto', md: 'initial' },
+        gap: { xs: '24px', md: '80px' },
+        padding: { xs: '40px', lg: '80px' },
       }}
     >
-      <Grid
-        container
-        spacing={2}
+      <Box
         sx={{
           order: { xs: 1, md: 0 },
         }}
       >
-        <Grid container spacing={2}>
+        <Grid
```

Comment:
Ditto here -- either use `Box` with `display: grid` or just use `Grid` (or Grid v2) and  don't change the `display`